docs/src/content/docs/components/using-components.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Using components
+description: Using components in MDX and Markdoc with Starlight.
+sidebar:
+  order: 1
+---
+
+Components let you easily reuse a piece of UI or styling consistently.
+Examples might include a link card or a YouTube embed.
+Starlight supports the use of components in [MDX](https://mdxjs.com/) and [Markdoc](https://markdoc.dev/) files and provides some common components for you to use.
+
+[Learn more about building components in the Astro Docs](https://docs.astro.build/en/core-concepts/astro-components/).
+
+## Using a component in MDX
+
+You can use a component by importing it into your MDX file and then rendering it as a JSX tag.
+These look like HTML tags but start with an uppercase letter matching the name in your `import` statement:
+
+```mdx
+---
+# src/content/docs/example.mdx
+title: Welcome to my docs
+---
+
+import { Icon } from '@astrojs/starlight/components';
+import CustomCard from '../../components/CustomCard.astro';
+
+<Icon name="open-book" />
+
+<CustomCard>Components can also contain **nested content**.</CustomCard>
+```
+
+Because Starlight is powered by Astro, you can add support for components built with any [supported UI framework (React, Preact, Svelte, Vue, Solid, and Alpine)](https://docs.astro.build/en/core-concepts/framework-components/) in your MDX files.
+Learn more about [using components in MDX](https://docs.astro.build/en/guides/markdown-content/#using-components-in-mdx) in the Astro docs.
+
+## Using a component in Markdoc
+
+Add support for authoring content in Markdoc by following our [Markdoc set-up guide](/guides/authoring-content/#markdoc).
+
+Using the Starlight Markdoc preset, you can use Starlight’s built-in components with Markdoc’s `{% %}` tag syntax.
+Unlike MDX, components in Markdoc do not need to be imported.
+The following example renders Starlight’s [card component](/components/cards/) in a Markdoc file:
+
+```markdoc
+---
+# src/content/docs/example.mdoc
+title: Welcome to my docs
+---
+
+{% card title="Stars" icon="star" %}
+Sirius, Vega, Betelgeuse
+{% /card %}
+```
+
+See the [Astro Markdoc integration documentation](https://docs.astro.build/en/guides/integrations-guide/markdoc/#render-components) for more information on how to use components in Markdoc files.
+
+## Built-in components
+
+Starlight provides built-in components for common documentation use cases.
+These components are available from the `@astrojs/starlight/components` package in MDX files and from the [Starlight Markdoc preset](/guides/authoring-content/#markdoc) in Markdoc files.
+
+See the components group in the sidebar for a list of available components and how to use them.
```

Comment:
```suggestion
See the components group in the sidebar for a list of available components and how to use them.
```
"the components group" in the sidebar feels off to me. If someone is on this page, they'll already be in this section in the sidebar. And, I know "group" is the way we define the sidebar sections, but I don't think someone reading/navigating a page thinks of "group" as identifying where they are. Heading? Section maybe?

I don't know the history of how this line came to be, so I'm hesitant to make a suggestion that might have been overruled previously. Just calling it out! I at first for a moment didn't realize you were pointing me to look in the sidebar, I thought there was some "component group" as a thing somewhere I needed to find. So I think the terminology just feels a little non-standard for someone *reading* this docs site, as opposed to building one.

Reply:
No big history! The vague inspiration was this in the getting started guide:

> See the guides in the sidebar for help adding content and customizing your Starlight site.

But no “group” mention there either.

Reply:
Would saying "See the sidebar for a list of available components and how to use them." be too vague or confusing?

Reply:
I think that's just fine!

Reply:
Thanks a lot, just updated the wording 🌟